projects/smart-table/src/lib/components/column-selector/column-selector.component.scss

```diff
@@ -0,0 +1,10 @@
+li {
+  cursor: move;
+}
+::ng-deep .cdk-drop-list-dragging .cdk-drag {
+  transition: transform 250ms cubic-bezier(0, 0, 0.2, 1);
+}
+/* Animate an item that has been dropped. */
+::ng-deep .cdk-drag-animating {
+  transition: transform 300ms cubic-bezier(0, 0, 0.2, 1);
```

Comment:
Is er een reden dat dit afwijkt van het standaard animatie gedrag? Anders moet dit `transform $animation-duration $animation-easing` zijn (geldt ook voor enkele regels hierboven).

Reply:
Dit is ondertss niet meer van toepassing